### PR TITLE
Branch realm

### PR DESCRIPTION
**Pull request by pkvarnwal**

### CODE REVIEW

File: _posts/2017-06-07-material-design [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+layout: post
```
Comment (aashishdhawan) [inline _posts/2017-06-07-material-design]: @pkvarnwal  this blog "Material design" is useless, please delete this file.

File: _posts/2017-07-09-Dynamic-Linking [hunk 2]
```diff
@@ -0,0 +1,69 @@
+---
```
Comment (aashishdhawan) [inline _posts/2017-07-09-Dynamic-Linking]: NOt publishing this dynamic link either
Please delete this too.

File: _posts/Realm-Doc [hunk 3]
```diff
@@ -0,0 +1,228 @@
+---
```
Comment (aashishdhawan) [inline _posts/Realm-Doc]: I can see spelling mistakes here. Please correct those first and submit again, I'll review after that.

File: _posts/2017-06-07-material-design (hunk 1 above)
Comment (pkvarnwal) [reply _posts/2017-06-07-material-design]: Boss no need to review the material design.
Only Realm blog.

**Pull request merged**